### ISSUE TITLE
Improve documentation for better contributor experience

### ISSUE THREAD

**Issue by Divy123** [publiclab/image-sequencer#1143]
- [ ] Providing more test based examples for better understanding of code and core API

**Comment by jywarren** [comment]
Hi! Perhaps we need a guide to writing tests (either UI or module) so as to
encourage this activity as well?

**Comment by Divy123** [comment]
I completely agree and I have already started exploring on the UI tests and will start adding that soon.

**Comment by Divy123** [comment]
Uploaded on GCI dashboard.

**Comment by SidharthBansal** [comment]
@Divy123 I think you are missing something. It will great if you can kindly visit guidelines for tasks and reformat all of your issues according to the guidelines.
Some of the missing things I observed are: Reserved for GCI tagline in BOLD, we use `beginner` instead of `easy`. If `beginner` then it should be in FTO template. I think this should be medium as documentation is a tough to piece to write for the first time at prod level, code links so that students can access the readme.md etc.


Thanks

**Comment by Divy123** [comment]
Done here!!

**Comment by SidharthBansal** [comment]
Please add code links here and add repo name in the task at GCI dashboard.
Then kindly hide unwanted comments and ping me again. Thanks

**Comment by SidharthBansal** [comment]
Published on GCI dashboard

**Comment by syobonpastel** [comment]
I want to do this task after my task is finished. But I couldn’t understand what to do in this task. What kind of skills will I need? Also what should I do in this task? Thanks.
